Cargo.toml

```diff
@@ -69,12 +69,14 @@ serde_cbor = "0.11.1"
 getrandom = "0.2.1"
 
 # Async
-tokio = { version = "1.0", features = ["rt-multi-thread"] }
+tokio = { version = "1.0", features = ["rt-multi-thread", "macros"] }
```

Comment:
It looks like `macros` is only for `tokio::test`? You could add it under `dev-dependencies` so they're only built when running the tests then.

Reply:
good point, updated it. 

( I actually wasn't sure how to enable a feature just for `dev-depencencies`, but `tokio::test` is a re-export from `tokio-macros` anyways.

Reply:
You can just have the same dependency line with a different set of `features`, it will merge the sets together when building the tests, but otherwise only use the features from the main dependency.